kafka/config.go

```diff
@@ -91,13 +93,29 @@ func value2string(v ConfigValue) (ret string, errstr string) {
 		ret = fmt.Sprintf("%d", x)
 	case string:
 		ret = x
+	case types.Slice:
```

Comment:
Question: I didn't get where we're using this yet, or why we've added this case.

Reply:
If we set a slice as config value for a list type configuration

Reply:
ack